elements/pfe-jump-links/src/pfe-jump-links-nav.scss

```
@@ -0,0 +1,324 @@
+@import "../../pfe-sass/pfe-sass";
+
+
+$LOCAL: jump;
+
+$variables: (
+  Padding: calc(#{pfe-var(container-spacer)} * 2),
+  BackgroundColor: pfe-color(surface--base),
+);
+
+:host {
+  //Default styles
+  // font-family: 'Red Hat Text', sans-serif;
+  // @include pfe-set-broadcast-theme(light);
+  // --theme: light;
+
+  // Example local variable, using BEM format
+  // --pfe-jump-links--Property: value;
+  font-weight: 400;
+  display: block;
+  position: sticky;
+  top: calc(#{pfe-var(container-spacer)} * 4);
+  padding: calc(#{pfe-var(container-spacer)} * 2);
+  --pfe-jump-links--BackgroundColor: BackgroundColor;
+}
+
+::slotted([slot="heading"]) {
+  margin: 0 0 1rem 0 !important;
+  margin-bottom: calc(#{pfe-var(container-spacer)} * 1);
+  font-size: 14px;
+  font-weight: 400;
+  text-transform: uppercase;
+}
+
+ul{
+  &.pfe-jump-links-nav {
+    &[horizontal] {
+
+    }
+    
+    padding: 1rem 1rem 1rem 0;
+    padding: 1rem 1rem 1rem 0;
+    margin: 0;
+    list-style: none;
+  }
+  &.sub-nav {
+    margin: 0;
+    padding: 0;
+    overflow-y: hidden;
+    transition: flex 1s linear;
+    display: flex;
+    flex-direction: column;
+    flex: 0;
+  }
+}
+
+li {
+  border-left: 3px solid transparent;
+  display: block;
+  text-decoration: none;
+}
+
+li.expand {
+  ul.sub-nav {
+    flex: 1;
+  }
+  a.sub-section {
+    display: table;
+    height: auto;
+  }
+}
+
+a {
+  display: table;
+  text-decoration: none;
+  font-size: 16px;
+  padding: 8px;
+  line-height: 1.5;
+  border-left: 3px solid transparent;
+  position: relative;
+  &.sub-section{
+    display: none;
+    // height: 0;
+    margin-left: 12px;
+    font-size: 14px;
+  }
+  &::after {
+    content: "";
+    position: absolute;
+    width: calc(100% - 4px);
+    height: calc(100% - 4px);
+    top: 0;
+    left: 0;
+  }
+  &:focus {
+    outline: none;
+
+    &::after {
+      border-radius: 3px;
+      border: 2px solid #62A1F4;
+    }
+  }
+}
+
+:host([default]) {
+  border: 1px solid #D2D2D2;
+  ::slotted([slot="heading"]) {
+    color: #151515;
+  } 
+  ul.pfe-jump-links-nav {
+    border-left: 1px solid #D2D2D2;
+  }
+
+  li{
+    &[active] {
+      border-left: 3px solid #0066CC;
+    }
+  }
+
+  a {
+    color: #72767B;
+    position: relative;
+
+    &:hover {
+      color: #151515;
+    }
+    &[active] {
+      color: #151515;
+    }
+    &::after {
+      content: "";
+      position: absolute;
+      width: calc(100% - 4px);
+      height: calc(100% - 4px);
+      top: 0;
+      left: 0;
+    }
+
+    &:focus {
+      outline: none;
+
+      &::after {
+        border-radius: 3px;
+        border: 2px solid #62A1F4;
+      }
+    }
+  }
+}
+
+:host([pfe-color="darkest"]) {
+  ::slotted([slot="heading"]) {
+    color: #CCCCCC;
+  } 
+  background-color: #151515;
+  border: 1px solid #72767B;
+  ul.pfe-jump-links-nav {
+    border-left: 1px solid #72767B;
+  }
+  li{
+    &[active] {
+      border-left: 3px solid #EE0000;
+    }
+  }
+  a {
+    color: #CCCCCC;
```

there are a variety of gray colors available: https://www.sassmeister.com/gist/4f095523377d120792295e321c73e01c

```suggestion
  a {
    color:   #{pfe-color(ui-complement--focus--on-dark)};
    &:hover {
       color: #{pfe-color(ui-complement--hover--on-dark)};
    }
    &[active] {
      color: #{pfe-color(ui-complement--on-dark)};
    }
```